test/conformance/README.md

```diff
@@ -19,4 +19,54 @@ all available devices/platforms, set 0. The default value is 1.
 If you run binaries for the tests, you can use the parameter
 `--platforms_count=COUNT` or `--devices_count=COUNT`.
 To set test device/platform name you want to run the tests on, use
-parameter `--platform=NAME` or `--device=NAME`.
\ No newline at end of file
+parameter `--platform=NAME` or `--device=NAME`.
+
+## Known failures
+
+The `UUR_KNOWN_FAILURE_ON` macro can be used to skip tests on devices where the
```

Comment:
Probably can be done in a separate task, but I'd like to see an XFAIL macro as well. Just so that we can catch when a test is fixed and re-enable it.

Reply:
https://github.com/oneapi-src/unified-runtime/issues/2548